setup.py

```diff
@@ -14,7 +14,7 @@
     install_requires=[
         'escapism',
         'python-slugify',
-        'jupyterhub>=0.9',
+        'jupyterhub>=0.9.1',
```

Comment:
Not sure this is what we want, but think it might be. See discussion in https://github.com/jupyterhub/kubespawner/pull/645#pullrequestreview-1140452434

```suggestion
        'jupyterhub>=1.3.0',
```